Bank_Account.rb

```diff
@@ -123,6 +137,7 @@ def self.find(id) #returns an instance of Account where the value of the id fiel
         end
```

Comment:
For the line above, do you think that ID should be stored as a string or an integer?